gatsby/src/components/ConferenceSpeaker.js

```diff
@@ -74,12 +72,8 @@ const SpeakerBio = styled.div`
 `;
 
 export default function ConferenceSpeaker({ speaker }) {
-  const dateAndTime = returnDateTime(speaker.date_time, speaker.duration);
   return (
     <OnlineSeriesTalk>
-      <h3>
-        {dateAndTime.date} <span className="talk-time">{dateAndTime.time}</span>
-      </h3>
```

Comment:
I'm repurposing this component for featured speakers. We don't need to show the talk timeslot.

Reply:
Everything is still working and looks good.